app/(golems)/golems/components/Sidebar.tsx

```
@@ -0,0 +1,147 @@
+        className="fixed top-4 left-4 z-50 md:hidden bg-[#1a1816] border border-[#e5950026] rounded-lg p-2 text-[#e59500]"
+        aria-label="Toggle navigation"
+      >
+        {mobileOpen ? <X className="h-5 w-5" /> : <Menu className="h-5 w-5" />}
+      </button>
+
+      {/* Mobile overlay */}
+      {mobileOpen && (
+        <div
+          className="fixed inset-0 bg-black/60 z-40 md:hidden"
+          onClick={() => setMobileOpen(false)}
+        />
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)